### ISSUE TITLE
Support allow-prereleases from Poetry

### ISSUE THREAD

**Issue by dbanty** [dephell/dephell#322]
### Required Information

Using dephell 0.7.8 with Python 3.7.5

### Description

Performing any dephell command with a pyproject.toml which contains an "allow-prereleases" tag (e.g. `black = {allow-prereleases = true, version = ">19"}`) strips that tag from the pyproject.toml.

[Poetry is deprecating](https://github.com/sdispater/poetry/commit/30c7b8ac0adbb673c21959fac5c7cbfc082830af) the older "allows-prereleases" in favor of "allow-prereleases" for consistency, so Black will have to support the new field.

**Comment by 515hikaru** [comment]
The report was reproduced. 

Suppose that pyproject.toml looks like this.

```toml
[tool.poetry.dependencies]
python = "^3.7"

[tool.poetry.dev-dependencies]
black = {version = ">19", allow-prereleases = true}
```

After exec `dephell deps add --from=pyproject.toml black`, pyproject.toml will be following.

```toml
[tool.poetry.dependencies]
python = "==3.*,>=3.7.0"
black = ">19"
```

If it has "allow**s**-prereleases" key, dephell won't remove the key.

BTW: `tool.poetry.dev-dependencies` is removed even though black is located in the dev-dependencies section. Is it another issue?

**Comment by orsinium** [comment]
Fixed in #323
